src/portal/src/app/config/config.component.ts

```diff
@@ -59,7 +59,7 @@ export class ConfigurationComponent implements OnInit, OnDestroy {
 
     public get hasAdminRole(): boolean {
         return this.session.getCurrentUser() &&
-            this.session.getCurrentUser().has_admin_role;
```

Comment:
Please define a model and use it in the get current user service to avoid use a backend properties in so many places.